packages/eslint-config-base/lib/modules/esm.js

```diff
@@ -9,5 +9,12 @@ module.exports = {
         'import/no-amd': 'error',
         // Report CommonJS require calls and module.exports or exports.*
         'import/no-commonjs': ['error', { allowConditionalRequire: false }],
+        // Order imports
+        'import/order': ['error', {
+            groups: ['builtin', 'external', 'parent', 'sibling', 'index'],
+            'newlines-between': 'always',
```

Comment:
I would either disable the new line or actually forbid it. Having new lines between groups is strange.

Reply:
Since it was suggested in the RFC I implemented it that way, but we can remove it for sure, I'm also not the biggest fan of forcing the newline after each group. @zebateira what do you think?

Reply:
Yeah that's fine, we can remove it. I think it brings structure to the import section but I'm fine if this is not enforced. It's very subjective as well.